sql/server.go

```diff
@@ -146,9 +146,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // exec executes the call. Any error encountered is returned; it is the
 // caller's responsibility to update the reply.
-func (s *Server) exec(call sqlwire.Call) error {
-	req := call.Args
-	resp := call.Reply
+func (s *Server) exec(req *driver.Request, resp *driver.Response) error {
```

Comment:
any reason for this being changed at the same time? we've also been switching to the xx(req)(reply, error) pattern. Should that be the case here too?